ci-operator/step-registry/optional-operators/configure-realtime-tuned-profile/optional-operators-configure-realtime-tuned-profile-commands.sh

```diff
@@ -4,12 +4,32 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+nodeRole=${APPLY_NODE_ROLE:=worker}
```

Comment:
can we please have snake style variables, see https://google.github.io/styleguide/shellguide.html#s7-naming-conventions

Reply:
Sure thing, updated